estree/estree.d.ts

```diff
@@ -0,0 +1,369 @@
+    object: Expression;
+    body: Statement;
+  }
+
+  interface SwitchStatement extends Statement {
+    discriminant: Expression;
+    cases: Array<SwitchCase>;
+    lexical: boolean;
+  }
+
+  interface ReturnStatement extends Statement {
+    argument?: Expression;
+  }
+
+  interface ThrowStatement extends Statement {
+    argument: Expression;
+  }
+
+  interface TryStatement extends Statement {
+    block: BlockStatement;
+    handler?: CatchClause;
+    finalizer?: BlockStatement;
+  }
+
+  interface WhileStatement extends Statement {
+    test: Expression;
+    body: Statement;
+  }
+
+  interface DoWhileStatement extends Statement {
+    body: Statement;
+    test: Expression;
+  }
+
+  interface ForStatement extends Statement {
+    init?: VariableDeclaration | Expression;
+    test?: Expression;
+    update?: Expression;
+    body: Statement;
+  }
+
+  interface ForInStatement extends Statement {
+    left: VariableDeclaration | Expression;
+    right: Expression;
+    body: Statement;
+  }
+
+  interface DebuggerStatement extends Statement {}
+
+  interface Declaration extends Statement {}
+
+  interface FunctionDeclaration extends Function, Declaration {
+    id: Identifier;
+  }
+
+  interface VariableDeclaration extends Declaration {
+    declarations: Array<VariableDeclarator>;
+    kind: string;
+  }
+
+  interface VariableDeclarator extends Node {
+    id: Pattern;
+    init?: Expression;
+  }
+
+  interface Expression extends Node {}
+
+  interface ThisExpression extends Expression {}
+
+  interface ArrayExpression extends Expression {
+    elements: Array<Expression | SpreadElement>;
+  }
+
+  interface ObjectExpression extends Expression {
+    properties: Array<Property>;
+  }
+
+  interface Property extends Node {
+    key: Expression;
+    value: Expression;
+    kind: string;
+    method: boolean;
+    shorthand: boolean;
+    computed: boolean;
+  }
+
+  interface FunctionExpression extends Function, Expression {}
+
+  interface SequenceExpression extends Expression {
+    expressions: Array<Expression>;
+  }
+
+  interface UnaryExpression extends Expression {
+    operator: UnaryOperator;
+    prefix: boolean;
+    argument: Expression;
+  }
+
+  interface BinaryExpression extends Expression {
+    operator: BinaryOperator;
+    left: Expression;
+    right: Expression;
+  }
+
+  interface AssignmentExpression extends Expression {
+    operator: AssignmentOperator;
+    left: Pattern | MemberExpression;
+    right: Expression;
+  }
+
+  interface UpdateExpression extends Expression {
+    operator: UpdateOperator;
+    argument: Expression;
+    prefix: boolean;
+  }
+
+  interface LogicalExpression extends Expression {
+    operator: LogicalOperator;
+    left: Expression;
+    right: Expression;
+  }
+
+  interface ConditionalExpression extends Expression {
+    test: Expression;
+    alternate: Expression;
+    consequent: Expression;
+  }
+
+  interface CallExpression extends Expression {
+    callee: Expression;
```

Comment:
@RReverser Yes, but the line:197 is `callee: Expression` not `callee: Expression | Super`.
Are you fixing it?